scripts/generate-pull-request-report.sh

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env bash
+
+# Github Client Credentials
+# Required to increase rate limits to a useable level.
+
+GITHUB_CLIENT_ID=cae8611e191547e163bb
```

Comment:
We probably want to externalize that?

Reply:
Agree that it is a bit weird, but since it's only used for the increased rate limit (5000 requests vs 60) and nothing else otherwise we could leave it for the sake of simplicity. 

If not, where do we move this?

Reply:
Makes sense, let's keep it this way.